src/main/java/mate/academy/spring/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Can't find this user.")
```

Comment:
It is better for further debugging to know what user wasn't found. It is better to have smth like this:
```suggestion
@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Can't find user by email: " + email)
```
but, I'm not sure this is possible to and dynamic data as a param to annotations

Reply:
@ResponseStatus has RetentionPolicy.RUNTIME
There is no way to dynamically generate a string used in an annotation. The compiler evaluates annotation metadata for RetentionPolicy.RUNTIME annotations at compile time, but GENERIC_GENERATED_NAME isn't known until runtime.